frontend/plan/src/components/selector/SectionDisplay.js

```diff
@@ -1,23 +1,63 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 
+import Badge from "../Badge";
+
 export default class SectionDisplay extends Component {
     stripTime = (s) => {
         let newS = s.replace(" to ", "-");
         newS = newS.replace("on", "");
         return newS;
-    }
+    };
+
+    getTimeString = (meetings) => {
```

Comment:
I understand that this is how we get the data from the Penn Registrar API, but I was wondering if it would be better if this was handled in the backend instead.

Reply:
Since the API will eventually be used for more than just PCP, I feel like we should try to keep most of the product-specific computation on the frontend. I do agree that the `SectionDisplay` component is getting pretty crowded, and there's probably an opportunity to refactor all of these `meeting` time utility functions, like getTimeString and finding conflicts, into a separate file.

Reply:
On second thought, that's actually the better move you're right. Especially since we do use the meeting times data in its current form from the API to generate the blocks in the schedule.